### PR TITLE
Add index.html, style.css and add linters

### PR DESCRIPTION
**Pull request by Kaiserabbas**
- edited the previous mobile version for responsiveness.
- created logo animation and text transitions for the main portfolio page.
- added CSS to make it responsive to different screen sizes.
- checked for linters errors and modified the style.css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+              </ul>
+            </section>
+      
+    <section class="about">
+      <div class="headline-skill-container">
+        <div class="about-headline">
+          <h2 class="heading-large">About Myself</h2>
+          <p class="headline-paragraph paragraph">
+            Hello I’m a software developer! I can help you build a product ,
+             feature or website. Look through some of my work and experience!
```
Comment (Olamarx) [inline index.html:167]: - [ ]  I will implore you to make the content width to be 40 and 60 percent just as it is for the design template.

### Figma design
image


### Your design
image

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @Kaiserabbas, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️
- Media query usage ✔️
- Desktop version ✔️
- Readme file professional ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [OPTIONAL] I see you created another repo, maybe because you were unable to fix the contact-form linter issue, I would encourage you to always maintain one repo so that your project won't be littered. I had to go check because I expected changes only on the file changes but I saw all the projects.
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kaiserabbas** [comment]
I have implemented the content width to be 40 and 60 percent just as it is for the design template.
Kindly approve. Thanks.
image

**Review by mckent05 — APPROVED** [review]
Hi @Kaiserabbas, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 
- Good job making your portfolio responsive on bigger screens 👏 

## Optional suggestions
**OPTIONAL** - Your PR title can be more descriptive it should be a summary of the tasks you have done in that PR, e.g `Responsive Desktop Design`, this would help make your repo more professional 👍 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**